packages/@lwc/engine/src/framework/base-lightning-element.ts

```diff
@@ -254,10 +247,9 @@ export interface LightningElement {
 function BaseLightningElementConstructor(this: LightningElement) {
     // This should be as performant as possible, while any initialization should be done lazily
     if (isNull(vmBeingConstructed)) {
-        throw new ReferenceError();
+        throw new ReferenceError('Invalid Constructor');
```

Comment:
=> `Illegal constructor`, the constructor is not invalid, the invocation of is is illegal.